### DISCUSSION TITLE
issues with DotSpatial.Positioning and System.ComponentModel.Primitives versions

### DISCUSSION THREAD

**Discussion by luigipavan** [DotSpatial/DotSpatial#1547] [General]
Hi, I'm new to gitHub.
I would like to experiment this library for GPS access.
I have downloaded the whole package and I'm trying to compile the demo but I get into some error  related to the version of some library.
Here it is:

Errore	CS1705	
L'assembly 'DotSpatial.Positioning' con identità 'DotSpatial.Positioning, Version=4.0.786.0, Culture=neutral, PublicKeyToken=null' usa 'System.ComponentModel.Primitives, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' la cui versione è successiva a quella dell'assembly 'System.ComponentModel.Primitives' a cui viene fatto riferimento con identità 'System.ComponentModel.Primitives, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'	

Sorry for the italian.
Any idea ?
Thanks in advance

Luigi

**Comment by luigipavan** [comment]
![Cattura_dotspatial](https://github.com/DotSpatial/DotSpatial/assets/96418905/0b39403c-cd00-446f-ad48-f1e1bd82fa9a)

**Reply by sindizzy** [reply]
You downloaded the code or the nuget package?
